src/client/containers/Channel/Channel.js

```diff
@@ -0,0 +1,24 @@
+import React, { useEffect, useState } from 'react';
+import { useParams } from 'react-router-dom';
+
```

Comment:
Please show in this file how to use the logged in users information. Fx Write it in p tag where you are already writing about the channel id